src/test/java/pageObjects/AuditTests.java

```diff
@@ -0,0 +1,43 @@
+package pageObjects;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AuditTests extends BaseTest {
+
+    @Test
+    public void successfullyAuditForCreateDevice() {
+        audit.successfullyCreateDeviceAudit();
+        String tableText = driver.findElement(By.className("MuiCollapse-wrapper")).getText();
+        Assert.assertTrue(tableText.contains("CREATE"));
+    }
+
+    @Test
+    public void successfullyAuditForAssignDevice() {
+        audit.successfullyAssignDeviceAudit();
+        String tableText = driver.findElement(By.className("MuiCollapse-wrapper")).getText();
+        Assert.assertTrue(tableText.contains("ASSIGN"));
+    }
```

Comment:
do we know to who the device is assigned to? if we do, we can also have a check to see if that persons name is displayed in the audit